mods/bones/init.lua

```diff
@@ -1,28 +1,31 @@
 -- Minetest 0.4 mod: bones
--- See README.txt for licensing and other information. 
+-- See README.txt for licensing and other information.
 
 bones = {}
+bones.enable = true -- enabled by default, can be changed by mods
 
 local function is_owner(pos, name)
 	local owner = minetest.get_meta(pos):get_string("owner")
-	if owner == "" or owner == name or minetest.check_player_privs(placer, "protection_bypass") then
+	if owner == "" or owner == name or minetest.check_player_privs(name, "protection_bypass") then
```

Comment:
I think you need to rebase. It has already been changed : https://github.com/minetest/minetest_game/commit/d90a0817223e33996b7fc3bb1c986da3a324aa98